<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_mod_3</class>
 <widget class="QDialog" name="fct_mod_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier les inscriptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_fct_mod_3">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>numSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>prenomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pays</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>categorieSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>numEq</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_nom"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Prenom:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_prenom"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_modifier">
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_mod_3">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_modifier</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_3</receiver>
   <slot>modifier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_3</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifier()</slot>
 </slots>
</ui>
